src/main/java/com/vagasti/backend/vaga/Vaga.java

```diff
@@ -23,16 +23,16 @@ public class Vaga {
   private String localizacao;
 
   @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL, mappedBy = "vaga")
-  private List<Inscricao> vagas;
+  private List<Inscricao> inscricao;
```

Comment:
É uma lista com vários itens, o nome deve estar no plural.

```suggestion
  private List<Inscricao> inscricoes;
```